releases/7.6.0.md

```diff
@@ -68,7 +68,8 @@ JavaScript:
 
```

Comment:
primevue seems major enough that if we're bumping to 3.51 let's update the list a few lines above here. (glad we're developing against latest, for sure!)